src/wrapped_radix_engine_toolkit/low_level/compiled_notarized_intent.rs

```diff
@@ -13,7 +13,7 @@ use crate::prelude::*;
     uniffi::Record,
 )]
 pub struct CompiledNotarizedIntent {
-    pub(crate) secret_magic: BagOfBytes,
```

Comment:
hide constructor, ensuring we always go via `TryFrom<ScryptoNotarizedTransaction>`